cmd/local/options.go

```diff
@@ -55,16 +63,31 @@ func componentsFromStrings(list []string) []keb.Components {
 		if len(s) >= 2 {
 			namespace = s[1]
 		}
-		components = append(components, keb.Components{Component: name, Namespace: namespace})
+		configuration := []keb.Configuration{}
+		for _, keyValue := range values {
+			splitKeyValue := strings.Split(keyValue, "=")
```

Comment:
Risky if the string includes a `=` in the value. Option to solve this could be to use a properties-reader for it (e.g. https://pkg.go.dev/github.com/magiconair/properties)

Reply:
Fixed.